package.json

```diff
@@ -27,9 +27,9 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "boom": "^4.3.1",
-    "hapi": "^16.6.2",
-    "hoek": "^4.2.0",
-    "inert": "^4.2.1"
+    "boom": "^7.1.1",
+    "hapi": "^17.2.0",
+    "hoek": "^5.0.2",
```

Comment:
just a small thing: is hoek still needed? because you removed the assertion checks from index route. Not sure if it's used anywhere else